### PR TITLE
Add ARIA attributes used 

### PR DESCRIPTION
**Pull request by dawei-wang**
Added ARIA attributes used reference links 
This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html [hunk 1]
```diff
@@ -37,6 +37,12 @@ <h4 id="Working_Examples">Working Examples:</h4>
 <h3 id="Notes">Notes </h3>
 
 <h3 id="ARIA_attributes_used">ARIA attributes used</h3>
+<ul>
+	<li><a href="https://www.w3.org/TR/wai-aria-1.1/#radiogroup">radiogroup</a></li>
+	<li><a href="https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby">aria-labelledby</a></li>
+	<li><a href="https://www.w3.org/TR/wai-aria-1.1/#aria-checked">aria-checked</a></li>
```
Comment (hamishwillee) [inline files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html:43]: If it were me I'd link to the appropriate MDN page - e.g. https://developer.mozilla.org/en-US/docs/web/Accessibility/ARIA/Attributes/aria-checked and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role

The other two don't have MDN pages yet, but I'd link to where the pages will be.

Comment (dawei-wang) [reply files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html]: Done.

Comment (hamishwillee) [reply files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html]: Thanks very much! Looks good to me, but leaving to @estelle for final review as she's the expert.

Comment (dawei-wang) [reply files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html]: Thank you

File: files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html [hunk 2]
```diff
@@ -37,6 +37,12 @@ <h4 id="Working_Examples">Working Examples:</h4>
 <h3 id="Notes">Notes </h3>
 
 <h3 id="ARIA_attributes_used">ARIA attributes used</h3>
+<ul>
```
Comment (estelle) [inline files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html:40]: two are attributes and two are roles. Let's change the title to Related roles and attributes.

**Review by estelle — APPROVED** [review]
Thank you!

**Pull request merged**